src/abi/map.ts

```diff
@@ -647,4 +647,398 @@ export const MapFunctions: Map<string, AbiFunction> = new Map([
             },
         },
     ],
+    [
+        "replace",
+        {
+            name: "replace",
+            resolve(ctx, args, ref) {
+                // Check arguments
+                if (args.length !== 3) {
+                    throwCompilationError("replace expects two arguments", ref); // Should not happen
```

Comment:
this can actually happen for `m.replace(42)`

```suggestion
                    throwCompilationError("replace expects two arguments", ref);
```